src/main/java/Duke.java

```diff
@@ -43,16 +43,21 @@ private static void runDuke() {
             printUserPrompt();
             Command command = CommandManager.getCommand();
             printLineDivider();
-            for (String token : command.getCommandTokens()) {
-                if (isExiting(token)) {
-                    return;
-                }
-            }
+            if (exitIfContainExitCommand(command)) return;
```

Comment:
Should still wrap in curly brackets even though one line

Reply:
> Should still wrap in curly brackets even though one line

Ture! I used the refactor function and didn't pay attention to it, thanks for reviewing